orchestration/dagster_orchestration/hca_orchestration/resources/load_tag.py

```diff
@@ -0,0 +1,18 @@
+from dagster import resource, Field, StringSource, InitResourceContext, BoolSource
+from datetime import datetime
+
+
+@resource({
+    "load_tag_prefix": Field(StringSource),
+    "append_timestamp": Field(BoolSource),
```

Comment:
nit: we can just use `String` and `Boolean` (`Bool`? not sure)  here, the `Source` suffix is only important if we want to load their values from env vars